@here/harp-mapview/lib/MapView.ts

```diff
@@ -3498,6 +3498,7 @@ export class MapView extends EventDispatcher {
                 this.m_tileObjectRenderer.render(
                     tile,
                     zoomLevel,
+                    this.zoomLevel,
```

Comment:
With the current naming, it looks as if you were passing twice the zoom level here. I had to follow the code to understand what DataSourceTileList.zoomLevel means, which is actually **storageLevel**. Funny enough there's already **DataSourceTileList.storageLevel**, which is actually the floored zoom level (called in some VisibleTileSet.updateRenderList  _camera storage level_, which is a very confusing name).

I know the names were already there, but could you rename DataSourceTileList.storageLevel to DataSourceTileList.floorZoomLevel and DataSourceTileList.zoomLevel to DataSourceTileList.storageLevel? That would help readability a lot. I hope that doesn't force you to change too many files...

Reply:
I discussed this with Roberto on Monday, which is partly why this is still a draft.

Yeah, there are a lot of cases where zoomLevel is actually the storageLevel, @robertoraggi did you create a task?

Reply:
@atomicsulfate  @nzjony  Yes, I will create a task so we can address the naming.